core/controllers/topic_editor.py

```diff
@@ -99,17 +99,20 @@ def get(self, topic_id):
             raise self.PageNotFoundException
         topic_domain.Topic.require_valid_topic_id(topic_id)
 
+        start_cursor = self.request.get('cursor')
```

Comment:
If the cursor arg doesn't exist at all, should this be explicitly cast to None or an empty string? (Note that get() takes a second arg which is the default value.)

Add tests for this case too; it can happen.

Reply:
It is taken as an empty string. Will modify a test to check this also.

Reply:
Done.